dice_maiden_logic.rb

```diff
@@ -110,7 +110,7 @@ def check_wrath
 
 # Takes raw input and returns a reverse polish notation queue of operators and Integers
 def convert_input_to_RPN_queue(event, input)
-  split_input = input.scan(/\b(?:\d+[d]\d+(?:\s?[a-z]+\d+)*)|[\+\-\*\/]|(?:\b\d+\b)|[\(\)]/i)# This is the tokenization string for our input
+  split_input = input.scan(/\b(?:\d+[d]\d+(?:\s?[a-z]+\d*)*)|[\+\-\*\/]|(?:\b\d+\b)|[\(\)]/i)# This is the tokenization string for our input
```

Comment:
I think with this change it will cause a lot of other options to error out where they are expecting a number after the option. End users will start seeing a lot of errors like this: 

Roller encountered error with "10d10 r": Failed to match sequence (LABEL? PARTS) at line 1 char 1.

instead of just dropping the command for being invalid